### PR TITLE
Create maintainer_guide.mar.md

### PR DESCRIPTION
**Pull request by SoulNikhar**
maintainer guide in Marathi translation

### CODE REVIEW

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@SoulNikhar, Thanks for your contribution. Kindly fix the links. They seem wrong.

**Comment by SoulNikhar** [comment]
Hii ... @TenTraicion  
Now all the links are fixed. Please have a look.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Still, some links are broken.. kindly follow other translations for that. Thanks!

**Comment by SoulNikhar** [comment]
> Still, some links are broken.. kindly follow other translations for that. Thanks!

Sorry @TenTraicion, I followed maintainer_guide.hin.md and checked all the links from my side. If there are still any broken links, please provide me with screenshots so that I can look into it.

File: translations/maintainer_guide/maintainer_guide.mar.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# देखभाल करणाऱ्यांचे मार्गदर्शक
+
+हे मार्गदर्शक त्यांच्यासाठी आहे जे प्रकल्पात देखभालकर्ता म्हणून सामील होऊ इच्छितात, समुदायासाठी प्रकल्प व्यवस्थापित करण्यात आणि देखरेख करण्यात मदत करण्यासाठी. (हे **पहिल्यांदा योगदानकर्त्यांसाठी मार्गदर्शक नाही**)
+
+## भाषांतरे
+
+हे ट्यूटोरियल उपलब्ध आहे [इतर भाषांमध्ये](translation/README.md)
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.mar.md:7]:
```suggestion
हे ट्यूटोरियल उपलब्ध आहे [इतर भाषांमध्ये](/translation/README.md)
```

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
fixed and merged*

**Pull request merged**